packages/eslint-plugin/src/rules/index.ts

```diff
@@ -5,4 +5,6 @@ export default {
   "deprecated-imports": require("./deprecated-imports").default,
   "avoid-web-storage": require("./avoid-web-storage").default,
   "avoid-prisma-client-import-for-enums": require("./avoid-prisma-client-import-for-enums").default,
+  "disallow-prisma-include-only-true-not-field-selector":
+    require("./disallow-prisma-include-only-true-not-field-selector").default,
```

Comment:
why are we adding this?

Reply:
I'm very sorry for this, this is a commit from PR #13649, this branch got created from the earlier branch, instead of main.